backend/src/main/java/com/shipping/backend/controllers/ShippingRetrivalServiceController.java

```diff
@@ -0,0 +1,37 @@
+package com.shipping.backend.controllers;
+
+
+import com.shipping.backend.entities.PackageTypeResponse;
+import com.shipping.backend.services.ShippingRetrivalService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/ShippingService")
```

Comment:
Please read this article to follow the REST API Naming Conventions
https://restfulapi.net/resource-naming/
After you read it, please change your `Mappings` to a named convention please

Reply:
Mapping names updated